docs/community-toolkit/hosting-rust.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Rust hosting
+author: Alirexaa
+description: Learn how to use the .NET Aspire Rust hosting integration to host Rust applications.
+ms.date: 11/12/2024
+---
+
+# .NET Aspire Rust hosting
+
+[!INCLUDE [includes-hosting](../includes/includes-hosting.md)]
+
+[!INCLUDE [banner](includes/banner.md)]
+
+In this article, you learn how to use the .NET Aspire Rust hosting integration to host Rust applications.
```

Comment:
The intro paragraph needs to be a bit more detailed. It should link to Rust docs, and describe exactly what the value-proposition is that the hosting integration is providing. For inspiration, see the [intro paragraph here](https://review.learn.microsoft.com/en-us/dotnet/aspire/caching/stackexchange-redis-integration?branch=pr-en-us-2041&tabs=dotnet-cli&pivots=redis).